Setup/env_test.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/sudo /bin/bash
```

Comment:
Let's rename this file to:
shield_pre_install_check.sh

Reply:
OK. Done.